2021-08-19 23:01:20,897 [http-nio-8081-exec-6] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:01:21,219 [http-nio-8081-exec-6] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:01:21,224 [http-nio-8081-exec-6] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:01:21,263 [http-nio-8081-exec-6] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:04:22,234 [http-nio-8081-exec-8] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:08:00,732 [http-nio-8081-exec-8] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:08:00,737 [http-nio-8081-exec-8] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:08:15,894 [http-nio-8081-exec-8] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:08:26,826 [http-nio-8081-exec-10] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:08:55,869 [http-nio-8081-exec-10] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:08:55,874 [http-nio-8081-exec-10] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:09:06,906 [http-nio-8081-exec-10] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:09:20,281 [http-nio-8081-exec-2] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:10:11,121 [http-nio-8081-exec-2] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:10:11,127 [http-nio-8081-exec-2] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:10:24,083 [http-nio-8081-exec-2] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:10:28,282 [http-nio-8081-exec-4] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:11:06,598 [http-nio-8081-exec-4] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:11:06,602 [http-nio-8081-exec-4] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:11:10,231 [http-nio-8081-exec-4] DEBUG org.anatkor.dao.UserDao- SQLException during Query SELECT * FROM usr WHERE phone_number=? processing from class org.anatkor.dao.Utils.
2021-08-19 23:11:23,305 [http-nio-8081-exec-6] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
2021-08-19 23:11:31,279 [http-nio-8081-exec-6] INFO  org.anatkor.controllers.servlets.LoginServlet- User servlet called
